File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        temp = x
        s = 0
        while temp>0:
            s = (s * 10) + (temp%10)
            temp //= 10
        if x == s:
            return 1
        return 0
            

File: 9-palindrome-number/9-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int l = 0;
        int temp = x;
        
        
        while(x>0){
            l = l*10 + x%10;
            x =x / 10;
        }
        if(temp==l){
            return true;
        }
        return false;
    }
}